CHANGES.md

```diff
@@ -0,0 +1,8 @@
+## DC/OS 1.9.11
+
+### Notable changes
+* Updated to [DC/OS UI 1.9+v1.9.11-rc1](https://github.com/dcos/dcos-ui/tree/1.9%2Bv1.9.11-rc1)
```

Comment:
same as ee, we dont have this file yet 🤔

Reply:
I think added this file once contributes to the release process of 1.9.  Also the git template required changes to be reported on this file. 

We are maybe just the first to bump 1.9 after they decided to have the file, I think it would be cool for us to add it.

Reply:
There are no rules for that, and having this file right here is better than not having it. Still obviously needs to taken/read with a grain of salt.